go-controller/pkg/libovsdbops/acl.go

```diff
@@ -53,6 +53,19 @@ func FindACLsWithPredicate(nbClient libovsdbclient.Client, p aclPredicate) ([]*n
 	return acls, err
 }
 
+// GetACLs gets ACLs from the cache of the given ACL attributes (the given name and external ids,
+// or the given priority, direction, match and action
+func GetACLs(nbClient libovsdbclient.Client, acls []*nbdb.ACL) ([]*nbdb.ACL, error) {
```

Comment:
Sorry, I know I suggested the `GetACLs` name but now I think this should be more `FindACLs` since there is no guarantee that all the ACLs will be found so that there is no guaranteed 1:1 input to output relationship, I think in that case `Find` wording seems more appropriate.